src/App.jsx

```diff
@@ -1,54 +1,17 @@
 import React from "react";
-import EmailVerify from "./component/email/EmailVerify";
-import Home from "./molecules/Home";
-import EmptyWishlistViewProducts from "./pages/EmptyWishlistViewProducts";
-import WishlistViewProducts from "./pages/WishlistViewProducts";
-import AddNewProductPage1 from './AddNewProduct/pageOne/AddNewProductPage1'
-import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+
+import { BrowserRouter} from "react-router-dom";
+import PhoneNumberVerificationSuccessUI from "./molecules/modal/PhoneNumberVerificationSuccessUI";
+
+
 
 
 function App() {
   return (
     <BrowserRouter>
-      <Routes>
-        <Route
-          path="/"
-          element={
-            <>
-              <Home />
-              <Link to="/support">Help and Support</Link>
-              <br />
-              <Link to="/email-verification">Visit Email Verification Page</Link>
-              <br />
-              <Link to="/products">Add new product</Link>
-
-            </>
-          }
-        />
-        <Route
-          path="/email-verification"
-          element={
-            <>
-              <EmailVerify />
-            </>
-          }
-        />
-        <Route
-        path="/products"
-        element={<AddNewProductPage1/>}
-        />
-        <Route 
-            path="/" 
-            element={
-              <div className="p-6">
-              <h1 className="text-3xl font-bold text-red-600">Hello world!</h1>
-              <p>/wishlist (view products in wishlist)</p>
-              <p>/add-to-wishlist (add products to wishlist)</p>
-              </div>
-          } exact></Route>
-          <Route path="/wishlist" element={<WishlistViewProducts />}></Route>
-          <Route  path="/add-to-wishlist" element={<EmptyWishlistViewProducts />}></Route>
-      </Routes>
+
+      <PhoneNumberVerificationSuccessUI/>
+
```

Comment:
Great job designing the Phone Number Verification Success UI. However, add your components to the routes following the same convention as the codes you deleted. You can read the react-router dom documentation if you have any issue